src/backend/marsha/core/tests/test_api_xapi_statement.py

```diff
@@ -48,7 +48,7 @@ def test_xapi_statement_with_no_lrs_configured(self):
             content_type="application/json",
         )
 
-        self.assertEqual(response.status_code, 501)
+        self.assertEqual(response.status_code, 200)
```

Comment:
The docstring for this test should be updated according to the expected status code.